BirthdayGreetingsTest/src/utils/convertDate.js

```diff
@@ -23,9 +28,10 @@ export function convertDate(date) {
       if (checkRealDate(date)) {
         // Replace - with / and return the new date
         const newDate = date.replace(/-/g, "/");
-        console.log(`Successfully converted ${date} to ${newDate}`);
+        logger.info(`Successfully converted ${date} to ${newDate}`);
```

Comment:
Same as above you want to make logs easily searchable so it's best to separate the searchable message and the debug information. In this case the dates.
```js
logger.info({message: 'successfully_converted_date', from: date, to: newDate})
```